adaptive/runner.py

```diff
@@ -839,7 +839,8 @@ def default_save(learner):
         async def _saver():
             while self.status() == "running":
                 method(self.learner)
-                await asyncio.sleep(interval)
+                # No asyncio.shield needed, as 'wait' does not cancel any tasks.
```

Comment:
LGTM 👍 

Don't understand what this comment is adding though?

Reply:
There is another function named `wait_for`, which waits for the first task to complete and then cancels all the other ones.
If you want it not to cancel the other tasks you need to wrap them in `asyncio.shield`.

My comment is meant to say that this (the not cancelling) is indeed the behavior we want, and that `shield` is not needed in order to achieve this behavior.

Reply:
OK great. Thanks for clarifying! :)